jest/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for Jest 16.0.0
+// Type definitions for Jest 18.0.0
```

Comment:
since this is a major version update, can you please copy the files before the change and place them in `v16`, then make the change. 

See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/13744 for an example.